kotlin/src/main/kotlin/org/jdbi/v3/core/kotlin/Jdbi858Extensions.kt

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:Suppress("unused")
```

Comment:
I'd prefer to see unit tests that demonstrate usage of each of these methods, than to suppress an unused code error. Nothing fancy--just prove that using these methods looks and feels like idiomatic Kotlin.